linters/detect_secrets/detect_secrets_report.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const { getAnnotation } = require('./detect_secrets_annotations')
+const { countIssueLevels } = require('../../annotations_levels')
+
+/**
+ * Process detect-secrets output (generate annotations, count issue levels)
+ * @param {*} results detect-secrets json output
+ */
```

Comment:
Add information on return via @return

Reply:
I noticed the bandit_report and gosec_report don't have `returns` documentation.
If you approve the way I document the return value of this function I will add the same style for them.